mosfit/modules/engines/magnetar.py

```diff
@@ -23,22 +23,24 @@ def process(self, **kwargs):
         self._rest_t_explosion = kwargs['resttexplosion']
 
         Ep = 2.6e52 * (self._Mns / 1.4)**(3. / 2.) * self._Pspin**(-2)
+        # ^ E_rot = 1/2 I (2pi/P)^2, unit = erg
 
         tp = 1.3e5 * self._Bfield**(-2) * self._Pspin**2 * (self._Mns / 1.4)**(
             3. / 2.) * (np.sin(self._thetaPB))**(-2)
+        # ^ tau_spindown = P/(dP/dt), unit = s
+        # Magnetic dipole: power = 2/(3c^3)*(R^3 Bsin(theta))^2 * (2pi/P)^4
+        # Set equal to -d/dt(E_rot) to derive tau
 
         ts = [
             np.inf
             if self._rest_t_explosion > x else (x - self._rest_t_explosion)
             for x in self._times
         ]
 
-        # print(ts)
-        #
-        # raise SystemExit
-        #
-        luminosities = [Ep / tp / (1. + t * DAY_CGS / tp)**2 for t in ts]
+
+        luminosities = [2 * Ep / tp / (1. + 2 * t * DAY_CGS / tp)**2 for t in ts]
```

Comment:
Yeah these 2s are from the original 1971 paper that gave the equation - which is the only place I can verify that the timescale is derived correctly! So the 2s should stay